examples/example-dsl/src/main/scala/com/lightbend/kafka/scala/iq/example/serializers/AppSerializers.scala

```diff
@@ -6,22 +6,27 @@ package com.lightbend.kafka.scala.iq.example
 package serializers
 
 import models.LogRecord
-import org.apache.kafka.common.serialization.Serdes
+import org.apache.kafka.common.serialization.{ Serdes, Serializer, Deserializer, Serde }
 import io.confluent.kafka.serializers.AbstractKafkaAvroSerDeConfig
 import com.lightbend.kafka.scala.iq.serializers._
 
 trait AppSerializers extends Serializers {
   final val ts = new Tuple2Serializer[String, String]()
   final val ms = new ModelSerializer[LogRecord]()
   final val logRecordSerde = Serdes.serdeFrom(ms, ms)
-  final val tuple2StringSerde = Serdes.serdeFrom(ts, ts)
+
+  implicit val baSerde = byteArraySerde
+  implicit val tuple2StringSerde = Serdes.serdeFrom(ts, ts)
+  implicit val wss = windowedStringSerde
+  implicit val ss = stringSerde
+  implicit val ls = longSerde
```

Comment:
This is where we define the implicits that will automatically get converted to `Serialized`, `Produced`, `Consumed` etc. Mixin this trait with the application class.